DataFormats/L1THGCal/interface/ClusterShapes.h

```diff
@@ -0,0 +1,77 @@
+#ifndef CLUSTER_SHAPES_H
```

Comment:
hi @PFCal-dev - please use a less generic name for the include guard.Typical CMS convention is to have the subsystem and package in addition to the class name